docs/source/theory/dp-with-opendp.rst

```diff
@@ -33,6 +33,14 @@ us if you are interested in proof-writing. Thank you!
             >>> import opendp.prelude as dp
             >>> dp.enable_features("contrib")
             
+.. note::
+    In this documentation, we'll consistently refer to OpenDP Library functions
+    as "constructors", although they are not "constructors" as that term is used in Python:
+    class names which are called to create instances.
+
+    Instead, in OpenDP these "constructors" are regular functions which provide
+    a common interface across all supported languages,
+    and return wrappers around objects created in the Rust core.
```

Comment:
I think it's ok to swap constructor and function in the places you have, but the definition of a constructor is not quite right: In OpenDP parlance, a "constructor" is a function that returns a transformation or measurement. A function like `laplace_scale_to_accuracy` would not be a constructor, but `make_gaussian` would be.

Reply:
Thanks! Just pushed an update:

> Instead, in OpenDP these "constructors" are regular functions
> which return a transformation or measurement,
> and they provide a common interface across all supported languages.